### PR TITLE
Update event while moving from one calendar to another one

### LINKED ISSUE #4093: Calendar Event Modal doesn't update Calendar visually when moving event from one calendar to another one
### Steps to reproduce

1. Have two calendars that are not the "Personal" Calendar.
2. Move an event from one calendar to the other
3. The event changes it's calendar colour
4. check the modal again

### Expected behavior

The event modal calendar is the one I selected previously

### Actual behaviour

The event modal calendar is still the old calendar.

### Calendar app version

3.2.2

### CalDAV-clients used

Web

### Browser

Chrome

### Client operating system

Ubuntu

### Server operating system

Ubuntu

### Web server

Apache

### Database engine version

MySQL

### PHP engine version

PHP 8.0

### Nextcloud version

NC24 - dev

### Updated from an older installed version or fresh install

Updated from an older version

### List of activated apps

_No response_

### Nextcloud configuration

_No response_

### Web server error log

_No response_

### Log file

_No response_

### Browser log

_No response_

### Additional info

image

Destination was: /remote.php/dav/calendars/admin/work/127C6807-2E7C-444E-86EC-75224913EEC9.ics

### LINKED ISSUE #4655: Event calendar picker out of sync after moving event
### Steps to reproduce

1. Share calendar read-write from user A to B
2. Open the app as B
3. Create an event in A's shared calendar
4. Change the event title and move the event to B's personal calendar
5. Click on the event

### Expected behavior

The event shows B's personal calendar selected in the multiselect.

### Actual behaviour

It shows A's calendar in the multiselect (wrong). After page reload it shows B's personal calendar (correct).

### Calendar app version

4.0

### CalDAV-clients used

_No response_

### Browser

_No response_

### Client operating system

_No response_

### Server operating system

_No response_

### Web server

_No response_

### Database engine version

_No response_

### PHP engine version

_No response_

### Nextcloud version

_No response_

### Updated from an older installed version or fresh install

_No response_

### List of activated apps

_No response_

### Nextcloud configuration

_No response_

### Web server error log

_No response_

### Log file

_No response_

### Browser log

```Text
[ERROR] calendar: calendarObject is undefined 
Object { level: 0, app: "calendar", uid: "admin" }
ConsoleLogger.js:64
    log ConsoleLogger.js:64
    error ConsoleLogger.js:91
    eventSourceFunction eventSourceFunction.js:57
    _callee$ eventSource.js:101
    l dist.js:1
    _invoke dist.js:1
    O dist.js:1
    asyncGeneratorStep eventSource.js:1
    _next eventSource.js:3
    _asyncToGenerator eventSource.js:3
    _asyncToGenerator eventSource.js:3
    events eventSource.js:112
    unpromisify main.js:4685
    fetch main.js:6463
    fetchSource main.js:6100
    fetchSourcesByIds main.js:6088
    reduceEventSources main.js:6032
    _handleAction main.js:7070
    runTask main.js:6894
    drain main.js:6886
    drained main.js:6857
    tryDrain main.js:6839
    request main.js:6800
    request main.js:6872
    dispatch main.js:6969
    dispatch main.js:2833
    refetchEvents main.js:3208
    modificationCount CalendarGrid.vue:195
    later index.js:27
    (Async: setTimeout handler)
    debounced index.js:38
    VueJS 11
    incrementModificationCount calendarObjects.js:150
    wrappedMutationHandler vuex.esm.js:844
    commitIterator vuex.esm.js:466
    commit vuex.esm.js:465
    _withCommit vuex.esm.js:624
    commit vuex.esm.js:464
    boundCommit vuex.esm.js:409
    _callee2$ calendarObjects.js:288
    l dist.js:1
    _invoke dist.js:1
    O dist.js:1
    asyncGeneratorStep calendarObjects.js:1
    _next calendarObjects.js:3
    (Async: promise callback)
    asyncGeneratorStep calendarObjects.js:1
    _next calendarObjects.js:3
    _asyncToGenerator calendarObjects.js:3
    _asyncToGenerator calendarObjects.js:3
    updateCalendarObject calendarObjects.js:326
    wrappedActionHandler vuex.esm.js:851
    dispatch vuex.esm.js:516
    boundDispatch vuex.esm.js:406
    _callee5$ calendarObjectInstance.js:1815
    l dist.js:1
    _invoke dist.js:1
    O dist.js:1
    asyncGeneratorStep calendarObjectInstance.js:9
    _next calendarObjectInstance.js:11
    _asyncToGenerator calendarObjectInstance.js:11
    _asyncToGenerator calendarObjectInstance.js:11
    saveCalendarObjectInstance calendarObjectInstance.js:1849
    wrappedActionHandler vuex.esm.js:851
    dispatch vuex.esm.js:516
    boundDispatch vuex.esm.js:406
    _callee2$ EditorMixin.js:605
    l dist.js:1
    _invoke dist.js:1
    O dist.js:1
    asyncGeneratorStep EditorMixin.js:1
    _next EditorMixin.js:3
    _asyncToGenerator EditorMixin.js:3
    _asyncToGenerator EditorMixin.js:3
    save EditorMixin.js:619
```

### Additional info

_No response_

### PR DESCRIPTION
**Pull request by JuliaKirschenheuter**
Fixes #4093
Fixes #4655

### CODE REVIEW

File: src/store/calendarObjects.js [hunk 1]
```diff
@@ -164,18 +176,19 @@ const actions = {
 			logger.error('Old calendar Id and new calendar Id are the same, nothing to move …')
 			return
 		}
-
+		// newCalendarObject is NOT calendarObject but calendar!
 		const newCalendarObject = context.getters.getCalendarById(newCalendarId)
```
Comment (tcitworld) [inline src/store/calendarObjects.js:180]: In that case the variable might as well be simply renamed `newCalendar`?

**Review by ChristophWurst — APPROVED** [review]

**Pull request merged**